frontend/packages/component-library/src/spacer/Spacer.tsx

```diff
@@ -0,0 +1,37 @@
+import classNames from 'classnames';
+import {HTMLAttributes} from 'react';
+
+import type {ComponentSizeXSToL} from '@/common/types';
+
+import moduleStyles from './spacer.module.scss';
+
+export interface SpacerProps extends HTMLAttributes<HTMLDivElement> {
```

Comment:
also nit: for consistency would be nice to add(declare) a `className` prop here, since we also support `HTMLAttributes` props here, so with current implementation in case one will add a className to this component it won't work cause internal classNames wil overwrite the one from `HTMLAttributes`

Reply:
then it would be nice to remove it from proptypes (as HTMLAttributes<HTMLDivElement> add support of className prop as it's one of html attributes)